eslint/eslint-config/package.json

```diff
@@ -24,23 +24,23 @@
   ],
   "peerDependencies": {
     "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0",
-    "typescript": ">=3.0.0"
+    "typescript": ">=4.7.0"
```

Comment:
Is this actually necessary? Can we avoid making this a breaking change?

Reply:
This seems like a proper time to do it, and I don't know any way around it- if the consumer of this is using an older version of typescript then Eslint breaks